<?xml version="1.0"?>
<sdf version="1.6">
  <model name="pcd_terrain">
    <static>true</static>
    <link name="terrain_link">
      <collision name="terrain_collision">
        <geometry>
          <mesh>
            <uri>model://pcd_terrain/meshes/terrain.stl</uri>
            <scale>1 1 1</scale>
          </mesh>
        </geometry>
      </collision>
      <visual name="terrain_visual">
        <geometry>
          <mesh>
            <uri>model://pcd_terrain/meshes/terrain.stl</uri>
            <scale>1 1 1</scale>
          </mesh>
        </geometry>
        <material>
          <ambient>0.4 0.5 0.3 1</ambient>
          <diffuse>0.5 0.6 0.4 1</diffuse>
        </material>
      </visual>
    </link>
  </model>
</sdf>
